packages/hosted-form-v2/src/iframe-content/hosted-input-validator.ts

```diff
@@ -129,6 +130,10 @@ export default class HostedInputValidator {
         return string().max(200).required('Full name is required');
     }
 
+    private _getNoteSchema(): StringSchema {
+        return string().max(128, 'Payment note is too long');
+    }
```

Comment:
The final look on the manual order UI.
<img width="1037" alt="image" src="https://github.com/user-attachments/assets/3f982291-2f0c-4472-9cba-5db47b775848">